### ISSUE TITLE
documenation for setting up browser sync

### ISSUE THREAD

**Issue by redknite** [understrap/understrap#1001]
in the documentation site it says to change

	"browserSyncOptions": {
		"proxy": "localhost:8080/",
		"notify": false
	},

to my own site in the gulpfile.js.   I didn't see it in there, but it is in gulpconfig.json.  Am I supposed to add it to the gulpfile.js?  I tried changing the version in the .json file but it didn't seem to work propertly when I ran gulp watch-bs.   The site loads, but VERY slowly.  as in it takes about a minute to load.  I'm using local by flywheel for local development, and just going through the site is fine and extremely fast.

in my console it still shows localhost:3000 as the local url even though I changed it to mine "okaysite.local"  and my broswer console says "can't establish an connection to the server at ws::/localhost:3000/browswer-sync/socket.io[]"...

**Comment by axlright** [comment]
Yes you are correct, the browsersync address and other customization now lives in `gulpconfig.json`

Yes the browser URL will display as `localhost:3000` even though browsersync is proxying `okaysite.local` or just `localhost` or wherever you're doing your local development.

I don't add a trailing `/` at the end of my proxy addresses.

I don't have much experience with local development by flywheel. I use `gulp watch-bs` daily without the trouble you're experiencing.

Can you confirm you're running an up to date version of Browsersync - I think the repo is using 2.23.6 or similar.

-Alex

**Comment by Thomas-A-Reinert** [comment]
Thanks @axlright - your hints at least solved the case for me as I´m also developing on Local by Flywheel and I used to just forget about browsersync as it also took ages to load for me.
Loads lightning fast right now!

**Comment by axlright** [comment]
@Thomas-A-Reinert Wow! I didn't know what the hints were but I'm glad they worked! I seem to be the only person who doesn't use Flywheel. WP Engine just bought them but that's all I know about 'em.

**Comment by pattonwebz** [comment]
I hear good things about Flywheel but does not work on my OS :(

**Comment by redknite** [comment]
> 
> 
> Thanks @axlright - your hints at least solved the case for me as I´m also developing on Local by Flywheel and I used to just forget about browsersync as it also took ages to load for me.
> Loads lightning fast right now!

Hi @Thomas-A-Reinert .  what did you change?  I don't have a trailing slash and I am using 2.26.3

**Comment by redknite** [comment]
Here's what I found out.  it's that .local extension.  For some reason it's not very liked.  I changed to okaysite.rt in both my local setup and then in gulpconfig.json and all is working.

**Comment by pattonwebz** [comment]
There is a service that runs on some systems to convert `.local` TLDs to their IP addresses. Bonjour I think it is called. It may be conflicting on your system or it could possibly even be running on your router (my router runs it but maps .local domains to .home domains and it caused me a few issues at times).

**Comment by redknite** [comment]
makes sense.  Thanks.

**Comment by axlright** [comment]
FWIW My team does all local development on a url with `local-` at the beginning of the exact domain name the client is using. Makes it easy and works in all circumstances. So like if we were working for the New York Times (we're not) our local development addresses would be `https://www.local-nytimes.com`

Then we install SSLs locally in case we need to test payment gateways.

Anywhoo, I've heard similar difficulties of working with `.local` as a TLD

**Comment by Thomas-A-Reinert** [comment]
Hm, never heard of that .local problem. (Stupid me!)
Anyway, your notice solved my problem with Browser Sync and that´s all that counts for me, personally ;)

And I´m also glad that the original posters problem got solved. Another lesson learned.
Bonjour is Mac-dependant, right? (Another reason to hate Apple?) 

JFYI, @redknite : I just added
```
"browserSyncOptions": {
	"proxy": "tarcgn-20.local",
	"notify": false
},
```
without Protocoll, without trailing slash. And it really boosted it. On Windows 10. No way 'Bonjour' was a show-stopper.
